config_amboy.go

```diff
@@ -1,26 +1,57 @@
 package evergreen
 
 import (
+	"time"
+
+	"github.com/mongodb/amboy"
+	"github.com/mongodb/anser/bsonutil"
 	"github.com/mongodb/grip"
 	"github.com/pkg/errors"
 	"go.mongodb.org/mongo-driver/bson"
 	"go.mongodb.org/mongo-driver/mongo"
 	"go.mongodb.org/mongo-driver/mongo/options"
 )
 
+var (
```

Comment:
Nit: I think it is more readable to put the set of keys beneath the struct.

Reply:
Moved.